### DISCUSSION TITLE
Añadir en la carpeta `Layout` un `Base`

### DISCUSSION THREAD

**Discussion by colmedev** [UXCorpRangel/boilerplate#8] [Ideas]
Creo que debería ser añadido un `Base.astro` en la carpeta `Layout` para tener el contenido sin los elementos del footer o navbar por si alguien quiere hacer un 404

**Comment by felixicaza** [comment]
Me parece buena idea. 🙌

**Comment by felixicaza** [comment]
¡Solucionado @colmedev!
